docs/install/Knative-with-IKS.md

```diff
@@ -131,7 +125,28 @@ components, the recommended configuration for a cluster is:
     Make sure all the nodes are in `Ready` state. You are now ready to install
     Istio into your cluster.
 
-## Installing Istio
+With a Kuberntes cluster ready, you now have two choices on how to install
+Knative: via a one-click "add-on" or manually.
+
+## Installing Knative using an IKS managed add-on 
+
+The easiest way to install it is using the Managed Knative add-on facility.
+This one-click install process will install Knative, and Istio if not already
+installed, and provide automatic updates and lifecycle management of your
+Knative control plane.
+
+You can get the add-on via the "Add-ons" tab of your Kubernetes cluster's
+console page, or via the command line:
+```bash
+bx ks cluster-addon-enable -y knative $CLUSTER_NAME
```

Comment:
how do we get `bx` CLI ?

Reply:
thanks for this!  it should be `ibmcloud` not `bx` (that's old).